lib/core/Cargo.toml

```diff
@@ -49,6 +49,8 @@ hex = "0.4"
 reqwest = { version = "=0.11.20", features = ["json"] }
 electrum-client = { version = "0.19.0" }
 zbase32 = "0.1.2"
+x509-parser = { version = "0.16.0", features = ["verify"] }
+base64 = "0.22.1"
```

Comment:
Instead of adding a new dependency, I would just use one of the `base64`s re-exported by our dependencies where we need `base64`, like one of
```rust
// boltz_client::bitcoin::base64
// lwk_wollet::bitcoin::base64
```